llvm/lib/CodeGen/BranchRelaxation.cpp

```diff
@@ -718,6 +734,13 @@ bool BranchRelaxation::relaxBranchInstructions() {
     }
   }
 
+  // If we relaxed a branch, subsequent block offsets may be underestimated.
```

Comment:
nit: Call out explicitly that `adjustBlockOffsets run over all MBBs if any BB is relaxed`

Reply:
Fixed.